apps/native/app/src/components/offline/offline-icon.tsx

```diff
@@ -0,0 +1,43 @@
+import { impactAsync, ImpactFeedbackStyle } from 'expo-haptics'
+import { Image } from 'react-native'
+import { Navigation } from 'react-native-navigation'
+
+import styled from 'styled-components/native'
+import warningIcon from '../../assets/icons/warning.png'
+import { useOfflineStore } from '../../stores/offline-store'
+import { ComponentRegistry as CR } from '../../utils/component-registry'
+import { Pressable } from '../pressable/pressable'
+
+const Icon = styled(Image)(({ theme }) => ({
+  width: theme.spacing[3],
+  height: theme.spacing[3],
+}))
+
+export const OfflineIcon = () => {
```

Comment:
Note! This component is displayed in "navigation" drawers when topbar is disabled